src/nile/cli.py

```diff
@@ -233,8 +273,39 @@ def version():
 @network_option
 @click.option("--contracts_file", nargs=1)
 def debug(tx_hash, network, contracts_file):
-    """Locate an error in a transaction using contracts."""
-    debug_command(normalize_number(tx_hash), network, contracts_file)
+    """
+    Locate an error in a transaction using available contracts.
+
+    Alias for `nile status --debug`.
+    """
+    status_command(normalize_number(tx_hash), network, "debug", contracts_file)
+
+
+@cli.command()
+@click.argument("tx_hash", nargs=1)
+@network_option
+@click.option("--track", "-t", "watch_mode", flag_value="track")
+@click.option("--debug", "-d", "watch_mode", flag_value="debug")
+@click.option("--contracts_file", nargs=1)
+def status(tx_hash, network, watch_mode, contracts_file):
+    """
+    Get the status of a transaction.
+
+    $ nile status transaction_hash
+      Get the current status of a transaction.
+
+    $ nile status --track transaction_hash
+      Get (wait for) the final status of a transaction (REJECTED / ACCEPTED ON L2)
+
+    $ nile status --debug transaction_hash
+      Same as `status --track` then locate errors if rejected using local contracts
```

Comment:
```suggestion
      Same as `status --track` then locate errors if rejected using local artifacts
```

To keep consistency with Marto's suggestion.